### DISCUSSION TITLE
Disable `welcome` page

### DISCUSSION THREAD

**Discussion by jvelarde-mpi** [ONLYOFFICE/DocumentServer#1989] [Q&A] [answered]
Hello new to forum, I just updated my onlyoffice to latest version, but now it shows this "Test sample" page. Before I updated it said "document server ready".  How do I disabled this test sample ? thanks.

https://onlyoffice.modernparking.net/welcome/

**Comment by ShockwaveNN** [comment]
Hi, this page was introduced a long time ago, I think like 5 years ago or some

I think you can disable it via custom nginx config or via removing it altogether via removing it in `/var/www/onlyoffice/documentserver/` folder

**Comment by ghost** [comment]
/etc/onlyoffice/documentserver/nginx/includes/ds-docservice.conf

first row's there is a entry /welcome/, the part can be removed
```
#welcome page
rewrite ^/$ $the_scheme://$the_host/welcome/ redirect;
```

service nginx restart

**Reply by ShockwaveNN** [reply]
Thanks

**Reply by jvelarde-mpi** [reply]
this is what I see in the first few rows, I delete the first 2 rows but it says 502 Bad gateway nginx.

#welcome page
rewrite ^/$ $the_scheme://$the_host/welcome/ redirect;

#support old version
rewrite ^\/OfficeWeb(\/apps\/.*)$ $the_scheme://$the_host/7.2.1-34/web-apps$1 r$

#script caching protection
rewrite ^(\/web-apps\/apps\/(?!api\/).*)$ $the_scheme://$the_host/7.2.1-34$1 re$

#disable caching for api.js
location ~ ^(\/[\d]+\.[\d]+\.[\d]+[\.|-][\d]+)?\/(web-apps\/apps\/api\/document$
  expires -1;
  # gzip_static on;
  alias  /var/www/onlyoffice/documentserver/$2;
}

#suppress logging the unsupported locale error in web-apps
location ~ ^(\/[\d]+\.[\d]+\.[\d]+[\.|-][\d]+)?\/(web-apps)(\/.*\.json)$ {

**Comment by ghost** [comment]
Uppps sorry, did remove the line from all servers :D this was from memory

```
#welcome page
rewrite ^/$ $the_scheme://$the_host/welcome/ redirect;
```

can be removed

**Reply by jvelarde-mpi** [reply]
I remove those 2 lines, I restart service, now I get 502 bad gateway

https://onlyoffice.modernparking.net/

**Reply by ghost** [reply]
that should not be the problem... i did it few secods too. There must be another problem

**Reply by jvelarde-mpi** [reply]
after the update the NGINX started the using the "documentserver-example" instead of the "documentserver" directory, which is the reason am getting that sample test page.

**Comment by jvelarde-mpi** [comment, marked as answer]
I did the following steps and it works now: 

rm -rf /etc/onlyoffice/documentserver-example/  #Removes the whole unnessecary "example"
rm /etc/nginx/includes/ds-example.conf   #This will delete the link that stops nginx from starting with error like you mentioned
service nginx restart   #Should restart without problem
service supervisord restart  #As Example-Service is gone, it will re-register to only four active packets 
supervisorctl restart docservice   #Restart docservice and now you can see the green checkmark
